cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -184,3 +184,6 @@ spec:
     manageMachineDisruptionBudgets: false
     # Namespace in which to watch for the MachineDisruptionBudgets.
     machineDisruptionBudgetNamespace: openshift-machine-api
+  # If any of your topology labels or other input to the CRUSH map labels have the value `default`, you need to change this to avoid
+  # conflicts.
+  crushRoot: default
```

Comment:
Instead of adding `crushRoot` to the root of the cluster spec, how about add it under the `storage.config` section? These settings are just key-value pairs that would make sense for this setting.

Reply:
Could do, if you think that it makes more sense. I’m normally in favour of putting things in schema.

Reply:
Yes thanks, that's where it would be best for now. In retrospect, the entire `storage.config` was never ideal. One of these I'll rip it out and replace it with the strongly typed schema.